analysis/museumdraw/analysis1_nov25_vssAbstract.Rmd

```diff
@@ -187,57 +244,48 @@ joint[,colInd]=scale(joint[,colInd])
 ```
 
 ## Inferential statistics: Does recogizability of drawings increase with age?
-Fixed effects of age, object cateogry, number of strokes, draw duration, and avg intensity (e.g., amount of ink)
-Random effects of subject (session_id). Can't include random slopes (1+category|sessionId) since every child did not draw every category.
-```{r}
-## Full model first
-fullModel=lmer(normCorrect ~ age + category + mean_intensity + num_strokes + draw_duration + (1|session_id), joint, REML = FALSE)
-
-# Check for normality assumptions on residuals
-s=summary(fullModel)
-hist(s$residuals)
-shapiro.test(residuals(fullModel))
-qqnorm(s$residuals)
-
-# Full model minus age
-noAge=lmer(normCorrect ~ category + mean_intensity + num_strokes+ draw_duration + (1| session_id), joint, REML = FALSE)
-
-## Model testing - model that includes age is better than model without age.
-modelOut=anova(fullModel,noAge)
-pOut=format.pval(modelOut$`Pr(>Chisq)`[2],1); # these quotes make it hard to embed in text...shorter variable for ease
 
-## Refit model for with REML for reporting betas
-## This does mean that the model beta is hard to interpret relative to the raw data but I don't see a good way around this
-fullModelReport=lmer(normCorrect ~ age + category + mean_intensity + num_strokes + draw_duration + (1|session_id), joint, REML = TRUE)
-
-coefOut=coef(summary(fullModelReport))
-ageBeta=format(round(coefOut[2,1],2),2)
-ageSE=format(round(coefOut[2,2],2),2)
-ageZ=format(round(coefOut[2,3],2),2)
+``` {r}
```

Comment:
I think there was something wacky going on with your model structure. 

There's some complexity in this dataset because of the turker effects, but ignoring them for now, a simple model is `correct ~ age` in the fixed effects, with crossed item and subject effects. You can model those crossed effects with missing data, but (unlike your comment implied) you can't *nest* `category` within `session_id`. that nesting would imply that each child saw multiple instances of each category and so you would be fitting a separate category intercept for each child. same thing for nesting `age` within `session_id` - doing so would mean that a child had multiple ages - only for longitudinal data. :)

Reply:
Ah I see – modeling category as a random effect does make more sense; trying to model it as a fixed effect seems to be the cause of those convergence issues.  And happy to use glmer and not deal with data transforms – thanks for the clarifications!